l2rpn_baselines/DoubleDuelingDQN/train.py

```diff
@@ -98,8 +92,13 @@ def train(env,
 
 
 if __name__ == "__main__":
-    args = cli()
+    from grid2op.MakeEnv import make
```

Comment:
I prefer keeping the imports a the top of the file, unless they are in a try ctach

Reply:
None of these imports are needed for the code to work properly. It reduces import time by a massive amount to only import necessary code (at the top of the file) and import things needed for "main" only when required.
Also it is more stable, as the number of dependencies is reduced when not using the file as the "__main__". Don't you think ?

Reply:
I'd say it just impacts startup loading time: So <100ms on a train script that can run for several days is negligible.

Plus, you would have imported these anyways if you just use the `train` method, because it's first argument is `env`, that will can only be created by doing these grid2op imports. 

Since python doesn't reload modules if they are already in the namespace, I prefer keeping imports on top, just a personal preference tough (from C/C++ dev ;])